web/page0/src/test_apis.py

```diff
@@ -0,0 +1,330 @@
+                            {
+                                "name": "cvd-2",
+                                "build_source": default_build_source,
+                                "status": "done",
+                                "displays": [],
+                                "group_name": "group-1",
+                            },
+                        ],
+                    },
+                    {
+                        "name": "group-2",
+                        "cvds": [
+                            {
+                                "name": "cvd-1",
+                                "build_source": default_build_source,
+                                "status": "done",
+                                "displays": [],
+                                "group_name": "group-2",
+                            },
+                            {
+                                "name": "cvd-2",
+                                "build_source": default_build_source,
+                                "status": "done",
+                                "displays": [],
+                                "group_name": "group-2",
+                            },
+                        ],
+                    },
+                    {
+                        "name": "group-3",
+                        "cvds": [
+                            {
+                                "name": "cvd-1",
+                                "build_source": default_build_source,
+                                "status": "done",
+                                "displays": [],
+                                "group_name": "group-3",
+                            },
+                            {
+                                "name": "cvd-2",
+                                "build_source": default_build_source,
+                                "status": "done",
+                                "displays": [],
+                                "group_name": "group-3",
+                            },
+                        ],
+                    },
+                ],
+            },
+        ],
+    }
+}
+
+
+def gen_host_name(length):
+    return "".join(random.choices(string.ascii_uppercase + string.digits, k=length))
+
+
+def gen_operation_name(length):
+    return "".join(random.choices(string.ascii_lowercase + string.digits, k=length))
+
+
+def find_host(hosts, name):
+    if not hosts:
+        return None, -1
+
+    for idx, host in enumerate(hosts):
+        if host["name"] == name:
+            return host, idx
+    return None, -1
+
+
+def find_group(groups, name):
+    if not groups:
+        return None, -1
+
+    for idx, group in enumerate(groups):
+        if group["name"] == name:
+            return group, idx
+    return None, -1
+
+
+def init():
+    for zone in data["zones"]:
+        mem["zones"][zone] = data["zones"][zone]
+
+
+# GET /info
+@apis.route("/api/info", methods=["GET"])
+def info():
+    sleep(0.1)
```

Comment:
Does this sleep call is need? If this is for simulating the delay of server, delays by network would be sufficient.

Reply:
Checked that network delay is enough. (>200ms)
Removed all sleep calls.